winconf.pri

```diff
@@ -35,12 +32,15 @@ win32-g++* {
     DEFINES += _FILE_OFFSET_BITS=64
     DEFINES += __USE_W32_SOCKETS
 
+    QMAKE_CXXFLAGS += -std=c++17
+
```

Comment:
I would opt for `CONFIG += c++17`

Reply:
https://github.com/qbittorrent/qBittorrent/pull/14005#discussion_r546697684

Reply:
nevermind